style.css

```diff
@@ -880,3 +880,7 @@ iframe#launcher.zEWidget-launcher.zEWidget-launcher--active {
 .govuk-list> li:nth-child(1n+4) .article-link {
   display: none;
 }
+
+.promotion-banner {
+  color: #ffff  !important;;
```

Comment:
Being removed, Zendesk takes a while to update onto Sourcetree so I can upload changes

Reply:
The white hash value should be either 3 digits or 6. i.e, either #fff or #ffffff.